src/main/java/org/haffson/adventofcode/days/day01/Day01.java

```diff
@@ -55,7 +60,8 @@ public String secondPart() {
      *
      * @return the final frequency
      */
-    private int calculateFrequency() {
-        return 0;
+    private int calculateFrequency(int[] myIntArray) {
+        return Arrays.stream(myIntArray)
+                .sum();
```

Comment:
Wenn es nur ein so kurzes Statement ist, würde ich das ruhig auf einer Zeile lassen. Ist aber natürlich dir überlassen.

```suggestion
        return Arrays.stream(myIntArray).sum();
```